src/controllers/LandingController.ts

```diff
@@ -1,6 +1,11 @@
 import { Request, Response, NextFunction } from 'express';
+import { controller, httpGet } from 'inversify-express-utils';
 
+@controller('/')
 export class LandingController {
 
-    public renderView = (req: Request, res: Response, next: NextFunction) => res.render('landing');
+    @httpGet('')
```

Comment:
Out of curiosity is that empty string required or can that arg be omitted?

Reply:
It's required :(